### DISCUSSION TITLE
Why are the examples not working?

### DISCUSSION THREAD

**Discussion by xlassmgreat** [andersfylling/disgord#448] [Q&A]
I just used the most basic example from the list which is https://github.com/andersfylling/disgord/tree/master/docs/examples/ping-pong and it doesn't work. The bot isn't online or even ready. What is going on? It also has an ominous red cross instead of a tick. What is happening?

**Comment by andersfylling** [comment]
>  The bot isn't online or even ready. What is going on?

Most likely due to privileged intents. 
![image](https://user-images.githubusercontent.com/7851860/147413994-97384075-b95c-47a1-8b9e-a7c0bd216832.png)

Currently Disgord enables all intent, except those specified in Config.RejectEvents. In a future version, you will have to explicitly set events and/or intents that you need, which should avoid any confusion around this.

>  It also has an ominous red cross instead of a tick. What is happening?

Is this from another example?
